dev/reference/tools/download.md

```diff
@@ -10,7 +10,7 @@ This document collects the available downloads for most officially maintained ve
 
 ## TiDB-Binlog and Lightning
 
-If you want to download the latest version of [TiDB-Binlog](/reference/tools/tidb-binlog/overview.md) or [Lightning](/reference/tools/tidb-lightning/overview.md), directly download the TiDB package, because both TiDB-Binlog and Lightning are included in the TiDB package.
+If you want to download the latest version of [TiDB Binlog](/dev/reference/tidb-binlog-overview.md) or [Lightning](/dev/reference/tools/tidb-lightning/overview.md), directly download the TiDB package, because both TiDB-Binlog and Lightning are included in the TiDB package.
```

Comment:
```suggestion
If you want to download the latest version of [TiDB Binlog](/reference/tidb-binlog-overview.md) or [Lightning](/reference/tools/tidb-lightning/overview.md), directly download the TiDB package, because both TiDB-Binlog and Lightning are included in the TiDB package.
```